move-into-a-new-folder@anaximeno/files/move-into-a-new-folder@anaximeno/po/fi.po

```diff
@@ -0,0 +1,64 @@
+# SOME DESCRIPTIVE TITLE.
+# This file is put in the public domain.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: move-into-a-new-folder@anaximeno 1.1.0\n"
+"Report-Msgid-Bugs-To: https://github.com/linuxmint/cinnamon-spices-actions/"
+"issues\n"
+"POT-Creation-Date: 2024-03-02 20:05-0100\n"
+"PO-Revision-Date: 2024-04-02 19:40+0300\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3\n"
+"Last-Translator: Kimmo Kujansuu <mrkujansuu@gmail.com>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: fi\n"
+
+#. metadata.json->name
+#: text.py:13
+msgid "Move Into a New Folder"
+msgstr "Siirrä uuteen kansioon"
+
+#: text.py:15
+msgid "Name of the new folder:"
+msgstr "Uuden kansion nimi:"
+
+#: text.py:17
+msgid "New Folder"
+msgstr "Uusi kansio"
+
+#: text.py:20
+#, python-format
+msgid ""
+"Folder '%s' already exists inside the current directory, do you want to move "
+"the selected files inside?"
+msgstr ""
+"Kansio \"%s\" on jo hakemistossa. Haluatko siirtää valitut tiedostot sisään?"
```

Comment:
Just a note that there are a few instances of double quotes where the original uses single quotes. This is why the escaping happens in the translation.

Reply:
Help! Do I have to solve that " " thing somehow?

Reply:
Poedit should let you change the double quotes to single or I can push a change for each if you would like.

Reply:
I made the change using the github editor. Apparently it worked? Let me know if it didn't work. Thanks for letting me know - I wouldn't have noticed that myself.

Reply:
I'll take a look and then merge. Feel free to push any other edits you have in the meanwhile!